models/index.js

```diff
@@ -8,9 +8,11 @@ module.exports = function(sequelize) {
   models.AccessKey = sequelize.import(path.join(__dirname, 'AccessKey.js'));
   models.User      = sequelize.import(path.join(__dirname, 'User.js'));
   models.Feed      = sequelize.import(path.join(__dirname, 'Feed.js'));
+  models.Entry     = sequelize.import(path.join(__dirname, 'Entry.js'));
   models.AccessKey.sync();
   models.User.sync();
   models.Feed.sync();
+  models.Entry.sync();
   models.Feed.belongsTo(models.User, { foreignKey: 'owner_uid' });
   models.AccessKey.belongsTo(models.User, { foreignKey: 'owner_uid' });
```

Comment:
Need to add the association between a feed and feed entries:
'''Javascript
        Feed.hasMany(models.Entry, { foreignKey: 'feed_id' });
'''